### DISCUSSION TITLE
Why Scala is given version 2.12 but actually version 2.13

### DISCUSSION THREAD

**Discussion by LiJie20190102** [apache/kyuubi#6834] [Q&A] [answered]
https://github.com/apache/kyuubi/blob/eb1b5996c9b89de829e81d88a858c6f7b1fe9def/build/dist#L265-L266

**Comment by pan3793** [comment, marked as answer]
in short, we'd like to build an additional `kyuubi-spark-sql-engine_2.13-<version>.jar`, to make the Kyuubi binary tarball being Spark/Scala version-agnostic, this [blog](https://mp.weixin.qq.com/s/mxCKeOiVh33hH2-m1_g4Cw) should help you understand that.

PS: use plain text/hyperlinks as much as possible, instead of pictures, to refer to something
